src/sdp-utils.c

```diff
@@ -1119,7 +1119,7 @@ char *janus_sdp_write(janus_sdp *imported) {
 	if(!imported)
 		return NULL;
 	janus_refcount_increase(&imported->ref);
-	char *sdp = g_malloc(1024), mline[8192], buffer[512];
+	char *sdp = g_malloc(1024), mline[8192], buffer[2048];
```

Comment:
If the buffer is this large, now, then I guess the `sdp` allocation we start from should be larger too? We only reallocate it when appending m-lines, which means it should be large enough to handle global attributes before that.

Reply:
Mh I think it might be overkill, the vast majority of SDPs are never that large: I think we started from a lower value on purpose. I'll give this some thought and get back to you later on.

Reply:
To be more conservative, maybe let's keep the same diff that was before, so 2048+512=2560? It's still large, but not THAT large.